src/components/Profile/UserInfo/FollowModal.js

```diff
@@ -1,62 +1,61 @@
 import Button from "@mui/material/Button";
 import Modal from "@mui/material/Modal";
 import React from "react";
-import users from "../../TestData/users.json";
 import {Divider, Grid, List, ListItem, ListSubheader} from "@mui/material";
 import Typography from "@mui/material/Typography";
 import ImageText from "../../Reused/ImageText";
-import "../../Reused/reused.css"
+import "../../Reused/reused.css";
 
-const FollowModal = ({followers, user}) => {
-  const userData = users[user.email];
+const FollowModal = ({followers, data, loading}) => {
   const [open, setOpen] = React.useState(false);
 
   const handleOpen = () => setOpen(true);
   const handleClose = () => setOpen(false);
 
   let buttonLabel;
   if (followers) {
-    buttonLabel = `${userData.followers.length} follower`;
-    if (userData.followers.length !== 1) {
+    buttonLabel = `${data.length} follower`;
```

Comment:
Can probably do this with a ternary here, but once again being picky. It's fine if you want to leave it

Reply:
Agree, I implemented this